src/server/api/route.js

```diff
@@ -243,6 +269,14 @@ class RouteApi
             }
         }
     }
+
+    async createEinvoiceReceiveCapability(businessPartnerId) {
+        return await this.eventClient.publish('businessPartner.capabilities.eInvoice-receive.create', businessPartnerId);
```

Comment:
```suggestion
   createEinvoiceReceiveCapability(businessPartnerId) {
        return this.eventClient.publish('businessPartner.capabilities.eInvoice-receive.create', businessPartnerId);
```

No need to async/await if you're returning the promise.